### PR TITLE
Bump to 2.0.0a1

### PR DESCRIPTION
**Pull request by shanemcd**
Depends-On: https://github.com/ansible/ansible-runner/pull/650

### CODE REVIEW

**Review by Spredzy — APPROVED** [review]

**Comment by Spredzy** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

**Comment by Spredzy** [comment]
recheck

**Comment by pabelanger** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "ansible-runner"
-version = "2.0.0"
+version = "2.0.0a1"
 description = "Consistent Ansible Pythin API and CLI with container and process isoluation runtime capabilities"
```
Comment (kdelee) [inline pyproject.toml:4]: :eye: oops there is a typo there, that said not introduced here

**Comment by pabelanger** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

**Comment by shanemcd** [comment]
recheck

**Pull request merged**